backend/internal_assembler/section_table.ml

```diff
@@ -0,0 +1,73 @@
+module StringTbl = X86_binary_emitter.StringTbl
+module SectionTbl = Hashtbl.Make (X86_proc.SectionName)
+
+type t =
+  { mutable sections : Owee.Owee_elf.section list;
+    mutable num_sections : int;
+    section_tb : int SectionTbl.t;
+    mutable section_bodies : (int * bytes) list;
```

Comment:
This would probably be better using an auxiliary type `{ offset : int; body : bytes }` or something.